src/common/argscheck.js

```diff
@@ -36,6 +36,35 @@ function extractParamName (callee, argIndex) {
     return (/\(\s*([^)]*?)\s*\)/).exec(callee)[1].split(/\s*,\s*/)[argIndex];
 }
 
+/**
+ * Throws TypeError if the provided arguments do not have the expected types.
```

Comment:
```suggestion
 * checkArgs can be used to check the type of method parameters
 *
 * Throws TypeError if the provided arguments do not have the expected types.
```

Reply:
I like mine better 😅

Reply:
Mine is additional in front, as an introduction - felt that was missing.

Reply:
I'll try to incorporate it when I find some time